docs/Information_monitoring.rst

```diff
@@ -8,7 +8,7 @@ This information is exposed in two ways:
 * For every node, there is a web interface from Netdata itself at ``<node>.server.mila.quebec:19999``.
   This is accessible only when using the Mila wifi or through SSH tunnelling.
 
-  * SSH tunnelling: on your local machine, run ``ssh -L19999:<node>.server.mila.quebec:19999 -p 2222 login.server.mila.quebec``,
+  * SSH tunnelling: on your local machine, run ``ssh -L 19999:<node>.server.mila.quebec:19999 -p 2222 login.server.mila.quebec`` or ``ssh -L 19999:<node>.server.mila.quebec:19999`` if you have already setup your :ref:`SSH Login`,
```

Comment:
Would it be possible to wrap the line to 80 columns?

Reply:
I made the lines shorter, and the text a bit cleaner